browser/templates/browser/person.html

```diff
@@ -24,6 +24,10 @@
                 aria-hidden="true"
                 href="{% url "split-person" person.id %}">
             </a>
+            <a class="btn btn-lg fa fa-plus"
+                aria-hidden="true"
+                href="{% url "add-investigator" person.id %}">
```

Comment:
and let's use the flask icon (ideally it would be a flask with a plus, but ah well)